### DISCUSSION TITLE
Run command on successful deploy?

### DISCUSSION THREAD

**Discussion by keehun** [vercel/vercel#9004] [Help]
Is there a way to run a command when (and only when) a deployment succeeds? I haven't found anything unfortunately when searching for this.

**Comment by amyegan** [comment]
Hi @keehun! You may be able to use GitHub Actions to run a command after a successful deployment. Can you tell me more about what you are trying to do?

**Comment by keehun** [comment]
At build time, my build script needs to download a bunch of JSON files to incorporate into the build. My code repo itself contains a cache of these JSON files (too numerous to download fresh each time). So during build time, a script downloads only the JSON files which were updated since the last successful build. I need a way to update the "successful last build time". I don't want to advance this timestamp if the build failed, for example.
